src/test/scala/com/databricks/spark/csv/CsvSuite.scala

```diff
@@ -267,6 +309,7 @@ abstract class AbstractCsvSuite extends FunSuite with BeforeAndAfterAll {
       .withDelimiter(',')
       .withQuoteChar(null)
       .withUseHeader(true)
+      .withNullValue("")
```

Comment:
I guess this one is not related with this PR. The default value will be already `""`.